crates/cli/src/main.rs

```diff
@@ -358,11 +357,7 @@ fn main() -> io::Result<()> {
             Ok(format_exit_code)
         }
         Some((CMD_VERSION, _)) => {
-            print!(
-                "{}",
-                concatcp!("roc ", include_str!("../../../version.txt"))
-            );
-
+            println!("roc {}", VERSION);
```

Comment:
I added the `roc` prefix to this command's output to match the output of `roc --version`